protozfits/digicam.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from astropy.utils.decorators import lazyproperty
+
+from .simple import File
+from .patch_ids import PATCH_ID_INPUT, PATCH_ID_OUTPUT
+
+
+def event_source(path, run_id=0):
+    for event in File(path).Events:
+        yield Event(event, run_id)
+
+
+class Event:
+    def __init__(self, event, run_id):
+        self.run_id = run_id
+        self._event = event
+
+        _e = self._event
+        _w = self._event.hiGain.waveforms
+
+        self.pixel_ids = _w.pixelsIndices
+        self._sort_ids = np.argsort(self.pixel_ids)
```

Comment:
Can we take advantage of the fact that the pixel_ids can be sorted once only? 
I suppose one should do this each when we read a new file.

Reply:
I think we can take advantage of that, but what if the pixel mapping changes in the middle of the file?

Reply:
Yes but in principle it should not. Should we ask @Etienne12345 ?

Reply:
okay .. so .. does this mean, these pixel ids might in the future go into the event header?